MAUI/TreeMap/events.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: Events in .NET MAUI TreeMap (SfTreemap) control | Syncfusion
+description: Learn here all about the Events support in the Syncfusion .NET MAUI TreeMap (SfTreemap) control and more.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+
+# Events in the .NET MAUI TreeMap (SfTreeMap)
+
+## SelectionChanged
+
+The `SelectionChanged` event occurs when there's a change in the selection within a tree map item.
+Below is a list of the arguments:
+
+* `Sender`: This contains the `SfTreeMap` object.
+
+* `SelectionChanged`: This event occurs when a selection change action is performed on an element available in the `TreeMapSelectionChangedEventArgs` object.
```

Comment:
We can get the below details from the `SelectionChanged` event.

    * `OldItems`: Returns the previously selected item.
    * `NewItems`: Returns the newly selected item.
    * `Category`: Returns the hierarchical path of the selected item within its groups.
    * `IsSelected`: Returns the items that are selected or deselected.
    * `GroupLevel`: Returns the level of grouping for the selected items.

Reply:
Updated as you mentioned